pages/12.Plugins/21.Twilio/docs.en.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Twilio
+media_order: ''
+body_classes: ''
+order_by: ''
+order_manual: ''
+taxonomy:
+    category:
+        - docs
+---
+
+-------------------
+## Mautic - Twilio plugin
+
+
+Before you start to send text messages from your Mautic, it needs to be connected to the service which can send them. The first and default implemented service is [Twilio][twilio]. In order to configure the text messages correctly, follow these steps:
+
+1. Create an account at Twilio.com.
```

Comment:
I would hyperlink the URL here just to make it easier for people, using the existing link macro